content/v2.0-preview/get-started/get-started-with-managed-resources.md

```diff
@@ -188,44 +189,42 @@ spec:
 EOF
 ```
 
-The {{< hover label="xr" line="2">}}apiVersion{{< /hover >}} and 
-{{< hover label="xr" line="3">}}kind{{</hover >}} are from the provider's CRDs.
-
-
-The {{< hover label="xr" line="5">}}metadata.generateName{{< /hover >}} value is the 
-name of the created S3 bucket in AWS.  
-This example uses the generated name `crossplane-bucket-<hash>` in the 
-{{< hover label="xr" line="5">}}$bucket{{</hover >}} variable.
+The {{< hover label="xr" line="6">}}metadata.generateName{{< /hover >}} gives a
+pattern that the provider will use to create a unique name for the bucket in S3.
+The generated name will look like `crossplane-bucket-<hash>`.
 
-The {{< hover label="xr" line="8">}}spec.forProvider.region{{< /hover >}} tells 
-AWS which AWS region to use when deploying resources.  
-
-The region can be any 
-[AWS Regional endpoint](https://docs.aws.amazon.com/general/latest/gr/rande.html#regional-endpoints) code.
-
-Use `kubectl get buckets` to verify Crossplane created the bucket.
+Use `kubectl -n crossplane-aws-app get buckets.s3.aws.m.upbound.io` to verify Crossplane created the bucket.
 
 {{< hint type="tip" >}}
```

Comment:
```suggestion
{{< hint "tip" >}}
```

This is an existing issue, but I noticed these aren't actually supposed to have a `type=`. It just happens that when you do `type="tip"` it doesn't know what it should be and defaults to `tip`. So if you do e.g. `type="note"` you'll get a `tip` instead. 🤔

Reply:
ah good call out, there were 2 other `"note"` boxes on this page that were being displayed as tips. I'll update them all.